### DISCUSSION TITLE
Badge - "dot" option for all variants

### DISCUSSION THREAD

**Discussion by atumas-bananamilk** [mantinedev/mantine#6914] [Feature requests]
Currently the `<Badge />` component for "dot" variant only looks like the "default" variant (just with a dot):
![Screenshot 2024-09-29 at 15 13 06](https://github.com/user-attachments/assets/670dcfc8-2941-49cb-b569-eda4d7f774ec)

We'd want a "dot" on every single variant, example:
![Screenshot 2024-09-29 at 15 05 37](https://github.com/user-attachments/assets/d449396b-2482-477e-8af8-82d8007c05b9)

**Comment by rtivital** [comment]
You can do that by passing a circle icon to the left section of the Badge

**Reply by atumas-bananamilk** [reply]
Then what's the point of "dot" variant at all?
I could add circle icon on the "default" variant too.
We don't want to write any wrappers.
